packages/compiler/src/server/yaml-resolver.ts

```diff
@@ -35,6 +35,16 @@ export interface YamlScalarTarget {
    * The siblings of the target node
    */
   siblings: string[];
+
+  /**
+   * The children of the siblings of the target node
+   */
+  siblingsChildren: string[];
```

Comment:
Seems not something we should put here which can make things very complicated. Let's talk about this.